docs/tutorial/force_calibration/robust_fitting.rst

```diff
@@ -0,0 +1,146 @@
+.. _robust_fitting:
+
+Robust fitting
+--------------
+
+.. only:: html
+
+    :nbexport:`Download this page as a Jupyter notebook <self>`
+
```

Comment:
To be completely honest, we are currently still evaluating whether we should make the switch to this method, as it requires less work from FSEs in the field and it runs less risk of problems if these identified peaks shift or new ones appear.

It's a really good point though, we should already clarify the limitations / what we are going to do in the intro. How about the following?
```
So far, we have been using least-squares fitting routines for force calibration.
In that case, we assume that the error in the power at each frequency is distributed according to a Gaussian distribution.
:ref:`Blocking or windowing<blocking_windowing>` the power spectrum ensures that this assumption is close enough to the truth such that the fit provides accurate estimates of the unknown parameters.
Occasionally, the power spectrum might show a spurious noise peak.
Such a peak is an outlier in the expected behavior of the spectrum and therefore interferes with the assumption of having a Gaussian error distribution.
As a result, the fit is skewed. In those cases, it can be beneficial to do a robust fit.
When a robust fit is performed, one assumes that the probability of encountering one or multiple outliers is non-negligible.
By taking this into account during fitting, the fit can be made more robust to outliers in the data.
One downside of this approach is that the current implementation does not readily provide standard errors on the parameter estimates and that it leads to a small bias in the fit results for which Pylake has no correction.
Robust fitting can be used in combination with a least-squares fit, to identify outliers automatically, in order to exclude these from a second regular least-squares fit.
```